contract-api-reference/.eth-permanent-registrar/README.md

```diff
@@ -14,7 +14,7 @@ The registrar itself is called [BaseRegistrar](registrar.md). This contract impl
 * Name owners may reclaim ownership in the ENS registry if they have lost it.
 * Owners of names in the legacy registrar may transfer them to the new registrar, during the 1 year transition period. When they do so, their deposit is returned to them in its entirety.
 
-In addition, the registrar is an [ERC721](https://github.com/ensdomains/ens/blob/master/docs/ethregistrar.rst#id3) compliant nonfungable token contract, meaning that .eth registrations can be transferred in the same fashion as other NFTs.
+In addition, the registrar is an ERC721 compliant nonfungable token contract, meaning that .eth registrations can be transferred in the same fashion as other NFTs.
```

Comment:
You may also want to mention about NameWrapper which is ERC1155

Reply:
But technically the ERC1155 isn't in the .eth permanent registrar, right?

My understanding is that a name's ERC721 stays in the .eth registrar owned by the NameWrapper, then the NameWrapper mints a new ERC1155 to represent ownership.